packages/paste-core/components/display-heading/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {DisplayHeading} from '../src';
+
+describe('DisplayHeading', () => {
+  it('should render', () => {
```

Comment:
Suggestion(non-blocking): this test is kinda redundant because the tests below also verify that it renders.